lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js

```diff
@@ -9,7 +9,7 @@
         function (Backbone, _, gettext, TeamUtils, teamProfileHeaderActionsTemplate) {
             return Backbone.View.extend({
 
-                errorMessage: gettext("An error occurred. Try again."),
+                errorMessage: _.escape(gettext("An error occurred. Try again.")),
```

Comment:
Can you link to that documentation? It is my understanding that we should escape all user-provided strings (strings where we don't know what the text will be like, and could contain script tags), but I don't see the need to escape a plain text string that we are specifying.

@nedbat and/or @sarina, can you provide some context?